frostsnap_coordinator/src/frostsnap_persist.rs

```diff
@@ -28,15 +28,6 @@ impl Persist<rusqlite::Connection> for FrostCoordinator {
             [],
         )?;
 
-        conn.execute(
-            "CREATE TABLE IF NOT EXISTS fs_signing_sessions (
```

Comment:
we were not using this anywhere? We have `fs_signing_session_state`

Reply:
Yes seemed to have slipped through the cracks.